Sources/Data/Services/ExperienceStore/ExperienceStoreService.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ExperienceStoreService.swift
+//  RoverData
+//
+//  Created by Sean Rucker on 2018-05-03.
+//  Copyright © 2018 Rover Labs Inc. All rights reserved.
+//
+
+import Foundation
+import os
+
+class ExperienceStoreService: ExperienceStore {
+    func get(byID id: String) -> Experience? {
+        do {
+            guard let path = experienceLocalURL(id: id) else {
+                os_log("Not able to retrieve experience because local storage for it could not be determined.")
```

Comment:
All `os_log` calls need to have a "namespace" (forget the proper term) so they can be identified as Rover noise in the Console app when developers are using it for their applications.